fplc_method.py

```
@@ -0,0 +1,871 @@
+import json
+from datetime import datetime
+import sys
+import logging
+from PyQt6.QtWidgets import (
+    QMainWindow, QVBoxLayout, QHBoxLayout, QGroupBox, QTableWidget, QTextEdit,
```

Ruff (0.12.2)

`PyQt6.QtWidgets.QMainWindow` imported but unused

Remove unused import

(F401)